src/System.Private.ServiceModel/tools/test/Bridge/WcfTestBridgeCommon/CertificateGenerator.cs

```diff
@@ -424,8 +424,9 @@ private X509Crl CreateCrl(X509Certificate signingCertificate)
             
             _crlGenerator.Reset();
 
```

Comment:
Yeah, I see @roncain's point. We shouldn't need a single `_crlGenerator` given our changes nowadays; we should be able to simply new up a `new X509V2CrlGenerator();` to avoid the interference. 

That being said, the caller to `CrlEncoded` should be locked, as that call ultimately is going to be causing issues if > 1 test is trying to mutate the Crl at the same time.